src/App.js

```diff
@@ -36,8 +43,9 @@ function AppNavbar() {
           />{" "}
         </Link>
       </Navbar.Brand>
-      <Nav className="ml-auto text-muted">
-        {/* <div>Help 1200+ people</div> */}
+      <Nav className="ml-auto">
```

Comment:
this section is overflowing on iphone x, causing the nav bar to cover 2 lines

Reply:
Check this again with the new one?